docs/content/tutorials/quickstart.md

```diff
@@ -0,0 +1,242 @@
+
+## Prerequisites
+
+You will need:
+
+  * Java 7 or higher
+  * Linux, Mac OS X, or other Unix-like OS (Windows is not supported)
+  * 8G of RAM
+  * 2 vCPUs
+
+On Mac OS X, you can use [Oracle's JDK 
+8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) to install 
+Java.
+
+On Linux, your OS package manager should be able to help for Java. If your Ubuntu-
+based OS does not have a recent enough version of Java, WebUpd8 offers [packages for those 
+OSes](http://www.webupd8.org/2012/09/install-oracle-java-8-in-ubuntu-via-ppa.html).
+
+## Getting started
+
+To install Druid, issue the following commands in your terminal: 
+
+```bash
+curl -O http://static.druid.io/artifacts/releases/druid-0.9.0-bin.tar.gz
+tar -xzf druid-0.9.0-bin.tar.gz
+cd druid-0.9.0
+```
+
+In the package, you should find:
+
+* `LICENSE` - the license files.
+* `bin/` - scripts that will be used for this setup.
+* `conf/*` - template configurations for a clustered setup.
+* `conf-quickstart/*` - configurations for this quickstart.
+* `extensions/*` - all Druid extensions.
+* `hadoop-dependencies/*` - Druid Hadoop dependencies.
+* `lib/*` - all included software packages for core Druid.
+* `quickstart/*` - files useful for this quickstart.
+
+## Start up Zookeeper
+
+Druid currently has a dependency on [Apache ZooKeeper](http://zookeeper.apache.org/) for distributed coordination. You'll 
+need to download and run Zookeeper.
+
+```bash
+curl http://www.gtlib.gatech.edu/pub/apache/zookeeper/zookeeper-3.4.6/zookeeper-3.4.6.tar.gz -o $zookeeper-3.4.6.tar.gz
+tar xzf $zookeeper-3.4.6.tar.gz
+cd zookeeper-3.4.6
+cp conf/zoo_sample.cfg conf/zoo.cfg
+./bin/zkServer.sh start
+```
+
+## Start up Druid services
+
+With Zookeeper running, return to the druid-0.9.0 directory. In that directory, issue the command:
+
+```bash
+bin/init
+```
+
+Next, you can start up the Druid processes. This tutorial runs every Druid process on the same system. In production, 
+many of these Druid processes can be colocated even in a distributed cluster.
+
+```bash
+java `cat conf-quickstart/druid/historical/jvm.config | xargs` -cp conf-quickstart/druid/_common:conf-quickstart/druid/historical:lib/* io.druid.cli.Main server historical
+java `cat conf-quickstart/druid/broker/jvm.config | xargs` -cp conf-quickstart/druid/_common:conf-quickstart/druid/broker:lib/* io.druid.cli.Main server broker
+java `cat conf-quickstart/druid/coordinator/jvm.config | xargs` -cp conf-quickstart/druid/_common:conf-quickstart/druid/coordinator:lib/* io.druid.cli.Main server coordinator
+java `cat conf-quickstart/druid/overlord/jvm.config | xargs` -cp conf-quickstart/druid/_common:conf-quickstart/druid/overlord:lib/* io.druid.cli.Main server overlord
+java `cat conf-quickstart/druid/middleManager/jvm.config | xargs` -cp conf-quickstart/druid/_common:conf-quickstart/druid/middleManager:lib/* io.druid.cli.Main server middleManager
+```
```

Comment:
I guess most people trying this will know to put these each in background or run each in a different window or whatever, but it's tempting to cut/paste this whole thing to execute...